noir_stdlib/src/merkle/smt.nr

```diff
@@ -0,0 +1,73 @@
+use crate::merkle;
+
+struct SparseMerkleTree {
+    default_value: Field,
+}
+
+impl SparseMerkleTree {
+    /// Constrains that the new leaf is located in the new tree at an index
```

Comment:
we don't have doc comments in noir yet, only regular comments

Reply:
Now changed to regular comments